### PR TITLE
Cinder nfs backend 3.0

### PR DESCRIPTION
**Pull request by nicolasbock**
Backport to Cloud 6.

https://github.com/crowbar/crowbar-openstack/pull/469

### CODE REVIEW

**Comment by toabctl** [comment]
@nicolasbock we usually wait for 2 +1 comments before we merge things.

**Comment by nicolasbock** [comment]
Sorry about that. I only looked at the "passed tests" which was green. Doesn't that usually count the number of +1's?

File: chef/data_bags/crowbar/template-cinder.json [hunk 1]
```diff
@@ -131,7 +135,7 @@
     "cinder": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 47,
+      "schema-revision": 101,
```
Comment (Itxaka) [inline chef/data_bags/crowbar/template-cinder.json]: @nicolasbock Is this correct? As its a backport to cloud6, shouldnt it follow the cloud6 numbering and be 48?

Comment (toabctl) [inline chef/data_bags/crowbar/template-cinder.json]: @Itxaka right. the commit is wrong. Let's revert it and create a new PR.

**Comment by toabctl** [comment]
> Sorry about that. I only looked at the "passed tests" which was green. Doesn't that usually count the number of +1's?

I think there is the lgtm service which counts the +1's but tbh I wouldn't trust that. It also depends when somebody added the +1 (maybe before another force push or so).

**Comment by nicolasbock** [comment]
Yes, good point.

**Pull request merged**